cmd/gb-vendor/fetch.go

```diff
@@ -70,4 +99,5 @@ var FetchCmd = &cmd.Command{
 		}
 		return wc.Destroy()
```

Comment:
Yup, that will work, setup a defer to wc.Destroy then return
vendor.WriteManifest.

Also, there is no need to pack all these changes into one PR.

If you send me small PRs, I can review those faster and get them merged for
you.

On Wed, 27 May 2015 17:05 Dávid Kaya <notifications@github.com> wrote:

> In cmd/gb-vendor/fetch.go
> <https://github.com/constabulary/gb/pull/134#discussion_r31107953>:
>
> > @@ -70,4 +99,5 @@ var FetchCmd = &cmd.Command{
> >  		}
> >  		return wc.Destroy()
>
> @davecheney <https://github.com/davecheney> I've noticed that you are
> returning wc.Destroy() here. Wouldn't it be more intuitive if you return
> vendor.WriteManifest(...)? Since the wc.Destroy() is not the main objective
> of the command. Atleast I've used that approach in the update and delete
> command.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/constabulary/gb/pull/134/files#r31107953>.
>